index.php

```diff
@@ -2,13 +2,41 @@
     /* -*- coding: utf-8; mode: php; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:set fenc=utf-8 filetype=php et sw=4 ts=4 sts=4: */
     /* Copyright (c) 2007, 2014, The MacPorts Project. */
     include_once("includes/common.inc");
-    print_header('The MacPorts Project -- Home', 'utf-8');
+    print_header('The MacPorts Package Manager for macOS', 'utf-8');
 ?>
 
 
 <div id="content">
-
-    <h2 class="hdr">The MacPorts Project Official Homepage</h2>
+   <h2 class="hdr">MacPorts</h2>
+    <p>MacPorts is an open-source package management for macOS.</p>
+    <p>Latest MacPorts <a href="install.php">release: <b><?php print $macports_version_latest; ?></a></b></p>
+    
+    <h3 class="subhdr">Install MacPort</h3>
+     <ol>
+        <li>Install <a href="<?php print $guide_url . '#installing.xcode'; ?>">Xcode and the Xcode Command Line Tools</a></li>
+        <li>Agree to Xcode license in Terminal: <code>sudo xcodebuild -license</code></li>
+         <li>Download and Run the MacPorts installer <a href="<?php echo $catalina_pkg; ?>">MacPorts-Catalina.pkg</a> (<a href="install.php#installing">other OS version</a>).
+        </li>
+    </ol>
+    
+    <p>For more info and options see <a href="install.php">installation</a>.</p>
+    
+    <h3 class="subhdr">Install a Package</h3>
+    <p><code>sudo port install packagename</code></p>
+    
+    <p>Everything installed by MacPorts is located in <code>/opt/local</code> and keep macOS clean.</p>
+    
+    <h3 class="subhdr">Update instealled Packages</h3>
```

Comment:
"Installed Ports", not "instealled Packages". If we're going to use title case for headers, then "Installed" would be capitalized as well.